service/javadsl/api/src/main/java/com/lightbend/lagom/javadsl/api/Descriptor.java

```diff
@@ -390,14 +390,14 @@ public String toString() {
     /**
      * Describes a topic call.
      */
-    public static final class TopicCall<Message> {
+    public static final class TopicCall<MESSAGE> {
```

Comment:
Actually, no it doesn't.  Still, I'm not sure about the allcaps, it looks so ugly.  Like the topic call is shouting at me that it takes a message.